src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,86 @@
+.product-card {
+  box-sizing: border-box;
+  height: 406px;
+  width: 200px;
+  padding: 206px 16px 16px;
```

Comment:
The padding-top seems unusually large. This could be a sign that you're using padding to position the background image. It's better to handle the image positioning with the `background-position` property, which you've already used on line 9.